rfd/0000-rfds.md

```diff
@@ -101,3 +101,11 @@ The purpose of the `state` is to tell the reader whether they should care about
 this RFD at all. For example, `deprecated` RFDs can be skipped most of the
 time. `implemented` is relevant to Teleport users, but `draft` is mostly for
 Teleport engineers and early adopters.
+
+### Security
+
+Describe the security considerations for your design doc.
+
+### UX
+
+Describe the UX changes and impact of your design doc.
```

Comment:
Let's give some guidance here as well:

Explore UI, CLI and API user experience by diving through common scenarios that users would go through. 
When exploring, show CLI, UI and API requests and responses, users would observe
Consider making error messages actionable, explore common type of failure modes and how users can recover

Reply:
Done.